Makefile

```diff
@@ -1,5 +1,5 @@
-IMG ?= "crossplane/sample-stack-wordpress"
-VERSION ?= "0.1.0"
+include stack.env
```

Comment:
I think there is some duplicated logic here. Would you mind either deleting the existing `Makefile` or modify it?

Reply:
They're still both useful for now . . . `make` looks for `Makefile`, so it seems like we want that so that the structure matches the convention people are used to with `make`.

The reason I've added `stack.Makefile` is so that we can use `crossplane-cli` commands for building and publishing the stack, and so that the commands are the same as non-template stacks.

> I'm imagining this repo to be as lean as possible to showcase how easy it is

I agree with this. And having two makefiles does seem more complex than is strictly necessary. However, I'd like it to work with the `crossplane-cli` commands, so this is what I've come up with as a compromise. I'm open to suggestions of course for approaches that make sense for people and work with the crossplane cli. We could remove the `Makefile`, but then it'd only work with the `Dockerfile` directly and the crossplane cli, but maybe that'd be okay.

In the future, I imagine us improving the configurability of the `crossplane-cli` so that we could point it at a regular `Makefile`. The issue we are seeing here is that we have a single convention supported by the `crossplane-cli`, with no configurability or support for other conventions. Potential solutions include having the `crossplane-cli` look for a `Makefile` if a `stack.Makefile` doesn't exist.

The main reason the crossplane cli uses `stack.Makefile` at all is to play nicely with kubebuilder.

Reply:
Being able to use this project in `crossplane-cli` contexts will be helpful for demonstrations.

I'm also reluctant to bulk this project up with Jenkinsfiles that are not reusable by anyone wanting to use this project as an example project.  I would prefer a Travis, Circle, or similar.  That said, these Jenkinsfiles align with our current build process for other Stacks and we want to maintain builds of this project through the same pipelines.

I'd be in favor of merging now and renaming this to `stack-wordpress` (as an official stack) while creating a suite of lightweight sample stacks later: sample-stack-wordpress-go, sample-stack-wordpress-helm3, sample-stack-wordpress-kustomize, etc.  These samples would not include Jenkinsfile builds and their images would not need to be published.  The sample projects should include e2e (kind) tests that could be reused by anyone copying the patterns of this sample.  The sample projects could include multiple common build pipeline configurations: circle, travis, etc.

Reply:
I like the idea of having two classes of example.